app/src/main/res/layout/fragment_subscriptions.xml

```diff
@@ -28,17 +28,24 @@
             android:textColor="?android:attr/textColorSecondary"
             android:textSize="@dimen/text_size_small" />
 
-    <GridView
-            android:layout_below="@id/feeds_filtered_message"
-            android:id="@+id/subscriptions_grid"
-            android:layout_width="match_parent"
-            android:numColumns="3"
-            android:horizontalSpacing="2dp"
-            android:verticalSpacing="2dp"
-            android:layout_height="match_parent"
-            android:layout_gravity="center_horizontal"
-            android:paddingBottom="88dp"
-            android:clipToPadding="false"/>
+    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+        android:id="@+id/swipeRefresh"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_below="@id/feeds_filtered_message">
+
+        <GridView
+                android:id="@+id/subscriptions_grid"
+                android:layout_width="match_parent"
+                android:numColumns="3"
+                android:horizontalSpacing="2dp"
+                android:verticalSpacing="2dp"
+                android:layout_height="match_parent"
+                android:layout_gravity="center_horizontal"
+                android:paddingBottom="88dp"
```

Comment:
I am now unsure about that `paddingBottom` as it is needed for `clipToPadding`.
Also, please have a look about all other paddings to make sure that I did not make any mistakes :wink:

Reply:
`all_episodes_fragment.xml` now has a tiny white space on the bottom of the screen. I think the padding should be applied to the RecyclerView, not to the SwipeRefreshLayout, there.

The padding here in `fragment_subscriptions.xml` looks good.